modules/cert-manager-install-console.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * security/cert-manager-operator-install.adoc
+
+[id="cert-manager-install-console_{context}"]
+= Installing the {cert-manager-operator} using the web console
+
+You can use the web console to install the {cert-manager-operator}.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *OperatorHub*.
+
+. Enter *{cert-manager-operator}* into the filter box.
+
+. Select the *{cert-manager-operator}* and click *Install*.
+
+. On the *Install Operator* page:
+.. Select the *tech-preview* channel as the *Update channel*. The *tech-preview* channel installs the latest Technology Preview release of the {cert-manager-operator}.
+.. The *Installation Mode* is set to *All namespaces on the cluster (default)*. This mode installs the Operator in the Operator-recommended `openshift-cert-manager-operator` namespace to watch and be made available to all namespaces in the cluster.
+.. Choose the *Installed Namespace* for the Operator. The default Operator recommended namespace is `openshift-cert-manager-operator`.
++
+If the `openshift-cert-manager-operator` namespace does not exist, it is created for you.
+.. Click the *Enable Operator recommended cluster monitoring on the Namespace* checkbox to enable cluster monitoring for the Operator.
+.. Select an *Update approval* strategy.
++
+* The *Automatic* strategy allows Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available.
++
+* The *Manual* strategy requires a user with appropriate credentials to approve the Operator update.
+
+.. Click *Install*.
+
+.Verification
+
+. Navigate to *Operators* -> *Installed Operators*.
+. Verify that *{cert-manager-operator}* is listed with a *Status* of *Succeeded*.
+
+////
+Questions:
+ - When I click the operator after it's been installed, there are two "CertManager" tabs. One has an instance of "cluster", the other is empty.
```

Comment:
This will go away once [CM-12](https://issues.redhat.com/browse/CM-12) is done.

Reply:
@slaskawi Do we expect CM-12 to be addressed before this initial release goes out?

Reply:
I think so. @s-urbaniak ?